models/intermediate/int_jira.yml

```diff
@@ -455,7 +468,6 @@ models:
       - name: issue_day_id
         description: Unique ID hashed on `valid_starting_on` and `issue_field_history`.
```

Comment:
We can actually leave the test on this since it's hashed on source_relation as well. Could you revert and also update the description to include source_relation in the hash?

Reply:
Updated.

Reply:
Wait why does it say `issue_field_history` lol. Changed to `issue_id`.